src/pages/tasks/NewTaskDetailsPage.tsx

```diff
@@ -110,9 +104,12 @@ function NewTaskDetailsPage({task, route}: NewTaskDetailsPageProps) {
                         inputID={INPUT_IDS.TASK_TITLE}
                         label={translate('task.title')}
                         accessibilityLabel={translate('task.title')}
-                        value={taskTitle}
+                        value={Parser.htmlToMarkdown(Parser.replace(taskTitle))}
```

Comment:
NAB but maybe we should memoize this so we don't run the parser regex matching on the taskTitle for every render, even if it hasn't changed